packages/next/src/lib/get-project-dir.ts

```diff
@@ -35,7 +35,10 @@ export function getProjectDir(dir?: string) {
       Log.error(
         `Invalid project directory provided, no such directory: ${path.resolve(
           dir || '.'
-        )}`
+        )}`,
+        new Error().stack,
+        process.env.NEXT_PRIVATE_WORKER,
+        process.argv
```

Comment:
Is this stacktrace intentional? Looks like debugging 🤔

Reply:
Yeah looks left over cleaned up